jerry-core/include/jerryscript-core.h

```diff
@@ -471,6 +471,15 @@ typedef enum
   JERRY_ITERATOR_TYPE_SET,          /**< Set iterator */
 } jerry_iterator_type_t;
 
+/**
+ * JerryScript special Proxy object options.
+ */
+typedef enum
+{
+  JERRY_PROXY_SKIP_GET_CHECKS = (1u << 0), /**< skip [[Get]] result checks */
```

Comment:
It would be good to have a NULL value at the start. or something like JERRY_PROXY_ALL_CHECKS. The main reason would be to not have the "0" as a value which can disable things.